tests/improver-blend-adjacent-points/02_basic_mean.bats

```diff
@@ -44,7 +44,10 @@
 
   # Run nccmp to compare the output and kgo.
   improver_compare_output "$TEST_DIR/output.nc" \
-      "$IMPROVER_ACC_TEST_DIR/blend_adjacent_points/basic_mean/kgo.nc"
-  rm "$TEST_DIR/output.nc"
-  rmdir "$TEST_DIR"
+      "$IMPROVER_ACC_TEST_DIR/$KGO"
+
+  if [ -n "$RECREATE_BATS_KGO" ]; then
+    mkdir -p "$RECREATE_BATS_KGO/${KGO%/*}"
```

Comment:
Same as $(basename $KGO)?

Reply:
Yes, Tomek introduced us to this syntax.